assets/js/blocks/cart-checkout/cart/edit.js

```diff
@@ -164,6 +165,10 @@ const BlockSettings = ( { attributes, setAttributes } ) => {
  * Component to handle edit mode of "Cart Block".
  */
 const CartEditor = ( { className, attributes, setAttributes } ) => {
+	if ( attributes?.isPreview ) {
```

Comment:
optional chaining shouldn’t be needed here, attributes should always be available (as an empty object) you can also give it a default value `{}`.